app/components/sign-up.jsx

```diff
@@ -0,0 +1,325 @@
+                    <button onClick={e => methods.sendResetPassword()} className={classes.resetBtn}>
+                        Send Reset Password
+                    </button>
+                </div>
+            </div>
+        </div>)
+}
+
+export default React.forwardRef(SignUp);
+
+const useStyles = createUseStyles(theme => ({
+    SignUp: {
+        color: theme.colors.textPrimary,
+        margin: 0,
+        borderRadius: '1rem',
+        padding: '0',
+        fontFamily: 'Inter',
+        fontSize: '1rem',
+        fontWeight: '700',
+        lineHeight: '1rem',
+        textAlign: 'center',
+        // position: 'fixed',
+        overflowY: 'auto',
+    },
+    btnClick: {
+        color: theme.colors.primaryButtonBlue,
+        textDecoration: 'none',
+        background: 'none',
+        border: 'none',
+        '&:hover': {
+            color: theme.colors.mouseDownPrimeBlue,
+            cursor: 'pointer',
+        },
+    },
+    aLinkTerm: {
+        marginLeft: '0.3rem', // add space
+        marginRight: '0.3rem', // add space
+        textDecoration: 'none',
+        '&:hover': {
+            textDecoration: 'underline',
+            cursor: 'pointer',
+            color: theme.colors.mouseDownPrimeBlue,
+        },
+    },
+    tabs: {
+        width: '80%',
+        height: '3rem',
+        margin: 'auto',
+        borderRadius: '5rem',
+        border: '0.1rem solid #D9D9D9',
+        padding: '0.2rem 0.2rem 0 0.2rem ',
+        boxShadow: ' 0.3rem 0.3rem 1rem 0.3rem rgba(0, 0, 0, 0.1)',
+    },
+    tab: {
+        display: 'inline-block',
+        position: 'relative',
+        width: '50%',
+        color: theme.colors.colorPrimary,
+    },
+    tabSelected: {
+        display: 'inline-block',
+        position: 'relative',
+        borderRadius: '5rem',
+        background: '#DCE8F2',
+    },
+    btnContainer: {
+        width: '100%',
+        display: 'flex',
+        gap: '2rem',
+    },
+    btn: {
+        borderRadius: '.5rem',
+        border: '0.2rem solid',
+        borderColor: theme.colors.primaryButtonBlue,
+        backgroundColor: theme.colors.white,
+        color: theme.colors.primaryButtonBlue,
+        display: 'block',
+        margin: '1rem auto',
+        textAlign: 'center',
+        width: '50%',
+        '&:hover': {
+            backgroundColor: theme.colors.primaryButtonBlue,
+            borderColor: theme.colors.primaryButtonBlue,
+            cursor: 'pointer',
+            color: theme.colors.white,
+        },
+    },
+    btnClick: {
+        border: 'none',
+        background: 'none',
+        color: theme.colors.colorPrimary,
+        '&:hover': {
+            background: 'none',
+            borderColor: 'none',
+            textDecoration: 'underline',
+        },
+    },
+    btnClickSelected: {
+        color: theme.colors.primaryButtonBlue,
+    },
+    inputContainer: {
+        margin: 0,
+        padding: '2rem',
```

Comment:
padding: '2rem 0' - don't apply padding to the left/right. Let the component fill the width provided by the parent.